website/source/partials/builder/vsphere/iso/_NIC-not-required.html.md

```diff
@@ -0,0 +1,8 @@
+<!-- Code generated from the comments of the NIC struct in builder/vsphere/iso/step_create.go; DO NOT EDIT MANUALLY -->
```

Comment:
You need to add these as partials to the main vsphere doc file otherwise it won't appear in the documentation.

Reply:
Something like this in the vsphere-iso.html.md.erb? Or am I missing something.
```
### Network Adapter Configuration
<%= partial "partials/builder/vsphere/iso/NIC-required" %>
```

Reply:
Yep! That's right. This should be enough. 👍 
Don't forget the not required as well.
```
## Network Adapter Configuration
### Required
<%= partial "partials/builder/vsphere/iso/NIC-required" %>
### Optional
<%= partial "partials/builder/vsphere/iso/NIC-not-required" %>
```